src/loader/cmi.ml

```diff
@@ -512,7 +505,7 @@ let read_value_description env parent id vd =
   let open Signature in
   let id = Env.find_value_identifier env id in
   let container = (parent : Identifier.Signature.t :> Identifier.LabelParent.t) in
-  let doc = Doc_attr.attached container vd.val_attributes in
+  let doc, () = Doc_attr.attached Odoc_model.Semantics.Expect_none container vd.val_attributes in
```

Comment:
would it be useful to have a helper function do this? Since _most_ of the elements don't have internal tags.

Reply:
Done.